arrow-site/docs/docs/patterns/monads/README.md

```diff
@@ -365,318 +348,239 @@ Subsequently, `addStep` is called two times to transfer to `Conference` and then
 
 The name of this pattern is **Monad**.
 
-In Arrow terms, a Monad is an interface with two operations: A constructor `just`, and `flatMap`.
+In Arrow terms, a `Monad` can be formalized by the powers provided in an impl of the `Effect<F>` interface.
+The `Effect` interface defines the ability to have a delimited scope in which a continuation can be used to complete the context or yield a value.
```

Comment:
The concept of `continuation` may need an introduction, as it has loaded meaning.

Reply:
replaced for coroutine which anyone in Kotlin should already be familiar before they land here.